src/Fixer/Operator/AlignDoubleArrowFixerHelper.php

```diff
@@ -21,6 +21,10 @@
  * @author Carlos Cirello <carlos.cirello.nl@gmail.com>
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  * @author Graham Campbell <graham@alt-three.com>
+ *
+ * @internal
```

Comment:
adding `@internal` tag is bc breaker.
while I agree it would be a good move, we shall not. yet, as we deprecated it, it's better in long-term :+1: